Automattic/jetpack
# External Sources in Media Inserter: Hide the Google Photos Search via CSS

Fixes https://github.com/Automattic/dotcom-forge/issues/3240
Before:
![Captura de Pantalla 2023-07-27 a las 13 38 37](https://github.com/Automattic/jetpack/assets/1989914/3e86acc8-90f9-4ed7-abbd-19e530760e24)

After:
![googlephotosnosearch](https://github.com/Automattic/jetpack/assets/1989914/08bd7e86-0900-4fb3-9918-95bc49f43506)

## Proposed changes:

* We hide the search input for the Google Photos Media Inserter integration since Google Photos isn't searchable.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to a Jetpack-connected site.
* Edit or create a new Post.
* Open the Media Sidebar and open the Google Photos category.
* You should not see a search input.
* Open Pexels.
* You should see a search input field.
